src/call.c

```diff
@@ -1499,6 +1499,7 @@ int call_set_video_dir(struct call *call, enum sdp_dir dir)
 	if (!call)
 		return EINVAL;
 
+	call->estvdir = dir;
```

Comment:
Can this happen for `call_set_audio_ldir` too?

Reply:
The functions `call_set_..._ldir()` sets only the local direction. It does not generate a re-INVITE or UPDATE, nor change the media direction for the established state.

`call_set_video_dir()` should change the video direction by sending a re-INVITE or UPDATE and remember also the direction locally.